http/src/main/resources/reference.conf

```diff
@@ -70,7 +70,7 @@ dataproc {
   }
 
   # Cached dataproc 2.2.x image used by Terra
-  customDataprocImage = "projects/broad-dsp-gcr-public/global/images/leo-dataproc-image-2-2-52-debian12-2025-05-06-13-58-25"
+  customDataprocImage = "projects/broad-dsp-gcr-public/global/images/leo-dataproc-image-2-2-52-debian12-2025-06-10-18-31-09"
   # AN-503: Delete once AOU has switched to using Dataproc 2.2.X in prod
   # Cached dataproc 2.1.x image used by AOU
   legacyAouCustomDataprocImage = "projects/broad-dsp-gcr-public/global/images/leo-dataproc-image-2-1-11-debian11-2025-05-07-16-35-36"
```

Comment:
The one wrinkle here is that this is also a dataproc image that will expire. Looking at [AOU's image version](https://github.com/all-of-us/workbench/pull/9269), they're now using the most up-to-date AOU image so we can remove support for this legacy image

Reply:
Do you recommend removing it as part of this PR itself? Do you think we need to confirm anything with AoU/Owls team?